content/tutorials/grafana-fundamentals.md

```diff
@@ -14,42 +14,53 @@ Feedback Link: https://github.com/grafana/tutorials/issues/new
 
 In this tutorial, you'll learn how to use Grafana to set up a monitoring solution for your application.
 
-### What you'll learn
+### What you'll learn how to:
 
-- How to explore metrics and logs
-- How to build dashboards
-- How to annotate dashboards
-- How to set up alerts
+- Explore metrics and logs
+- Build dashboards
+- Annotate dashboards
+- Set up alerts
 
 ### What you'll need
 
 - [Docker](https://docs.docker.com/install/)
 - [Docker Compose](https://docs.docker.com/compose/)
+- [Git](https://git-scm.com/)
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Set up the sample application" %}}
 
 This tutorial uses a sample application to demonstrate some of the features in Grafana. To complete the exercises in this tutorial, you need to download the files to your local machine.
 
-In this step, you'll set up the sample application, as well as supporting services, such as [Prometheus](https://prometheus.io/), and [Loki](https://grafana.com/oss/loki/).
+In this step, you'll set up the sample application, as well as supporting services, such as [Prometheus](https://prometheus.io/) and [Loki](https://grafana.com/oss/loki/).
 
-- Clone the [example code](https://github.com/grafana/tutorial-environment) using [Git](https://git-scm.com/):
+- Clone the [Tutorial environment Git repository](https://github.com/grafana/tutorial-environment). 
+
+If using Git Desktop, then click the link above, click **Clone or download**, and then click [Open in Desktop](https://git-scm.com/).
+
+Or in the command line (`cmd` for Windows), enter:
 
 ```
 git clone https://github.com/grafana/tutorial-environment.git
 ```
 
-- Go to the directory where you cloned this repository:
+- Change to the directory where you cloned this repository:
 
 ```
+# Linux or Mac
 cd tutorial-environment
+
+# Windows example, replace the path with the one on your system
+
+cd "C:\GitFiles\tutorial-environment"
```

Comment:
I assume this is where GitHub Desktop puts the projects? Does it make sense to change this into something like: 

"Or, wherever your tool puts working copies"